not sure why CI is red:
```
Run echo "::add-matcher::.github/swiftformat.json"
Running SwiftFormat...
(lint mode - no files will be changed.)
Reading config file at /Users/runner/work/react-native-test-app/react-native-test-app/.swiftformat
Warning: /Users/runner/work/react-native-test-app/react-native-test-app/ios/ReactTestApp/ReactInstance.swift:1[15](https://github.com/microsoft/react-native-test-app/actions/runs/7957745599/job/21721190034?pr=1854#step:13:16):1: warning: (redundantReturn) Remove unneeded return keyword.
SwiftFormat completed in 0.44s.
Source input did not pass lint check.
1/15 files require formatting, 2 files skipped.
Error: Process completed with exit code 1.
```

did something change and now the lint throws errors?

--- a/example/package.json
+++ b/example/package.json
@@ -18,7 +18,7 @@
     "windows": "react-native run-windows --no-packager"
   },
   "dependencies": {
-    "@react-native-webapis/web-storage": "^0.2.0",
+    "@react-native-webapis/web-storage": "^0.2.1",
     "react": "18.2.0",
     "react-native": "^0.73.0",
     "react-native-macos": "^0.73.0",

--- a/ios/ReactTestApp/ReactInstance.swift
+++ b/ios/ReactTestApp/ReactInstance.swift
@@ -112,7 +112,7 @@ final class ReactInstance: NSObject, RNXHostConfig {
     // MARK: - RCTBridgeDelegate details
 
     func sourceURL(for _: RCTBridge) -> URL? {
-        return remoteBundleURL ?? bundleURL()
+        remoteBundleURL ?? bundleURL()
     }
 
     // MARK: - Private

--- a/package.json
+++ b/package.json
@@ -80,7 +80,7 @@
     "test:rb": "bundle exec ruby -Ilib:test -e \"Dir.glob('./test/test_*.rb').each { |file| require(file) }\""
   },
   "dependencies": {
-    "@rnx-kit/react-native-host": "^0.4.3",
+    "@rnx-kit/react-native-host": "^0.4.4",
     "ajv": "^8.0.0",
     "chalk": "^4.1.0",
     "cliui": "^8.0.0",

